app/assets/javascripts/discourse/app/templates/components/badge-card.hbs

```diff
@@ -4,7 +4,22 @@
 {{#if badge.has_badge}}
   <a href={{url}} class="check-display status-checked">{{d-icon "check"}}</a>
 {{/if}}
-<div class="badge-contents">
+{{#if isFavorite}}
+  {{d-button
+    icon="star"
+    class="favorite btn btn-default"
```

Comment:
btn class is already set by d-button IMO

Reply:
oh! good to know, thanks for the tip :)